frontend/src/AdminUserListView.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+/**
+ * A component that displays a list of all the messages in our application.
+ */
+class AdminUserListView extends React.Component {
+    constructor(props) {
+        super(props);
+
+        // This is a stateful component so we need to define a default representation of our state
+        this.state = {
+            text: '',
+            members: null
+        };
+    }
+
+    render() {
+        if (this.state.members == null) {
+            return (<div> loading members</div>);
+        }
+        return (
+            <div>
+                <div>
+                    <textarea value={this.state.text} onChange={this.onTextChange.bind(this)}></textarea>
+                    <button onClick={this.onButtonClick.bind(this)} disabled={this.state.text.length === ''}> submit </button>
+                </div>
+                <div className="member-container">
+                    {this.state.members.length === 0 && (<p> There are no members yet </p>)}
+                    {this.state.members.map((member) =>
+                        <div key={member.id} className="member-div">
+                            <div className="member-body">{member.email}</div>
+                        </div>
+                    )}
+                </div>
+            </div>
+        );
+    }
+
+    // componentDidMount is a very special function in React. It's part of a group of functions that form part of
+    // the React lifecycle. React calls each one of these functions to notify you the programmer that certain things
+    // have happened or are about to occur.
+    //
+    // Internally React will call componentDidMount after a component's output has been rendered to the DOM.
+    // Its common React practice to make any API calls in componentDidMount().
+    //
+    // Additional documentation about React's lifecycle can be found here:
+    // https://reactjs.org/docs/react-component.html#commonly-used-lifecycle-methods
+    componentDidMount() {
+       this.getMembers();
+    }
+
+    onTextChange(event) {
+        event.preventDefault();
+        this.setState({text: event.target.value});
+    }
+
+    onButtonClick(event) {
+        event.preventDefault();
+        fetch(`/messages?text=${this.state.text}`, {method: 'POST'})
```

Comment:
You'll want to change this to point to the servlet that creates users. I took a look at the backend code and it doesn't look like we don't have a servlet for this *specific* use case. 

We had originally decided to force the user to specify the user emails when creating a team but we changed that so that the admin can add users at will.  You'll have to create a relatively small servlet that creates a user given the email. It'll look similar to this:
```java

@WebServlet('/create-user')
public class CreateUserServlet {
    private UserDatastore userDatastore;

    public void init() {
        userDatastore = new UserDatastore();
    }

   public void doPostServletRequest request, HttpServletResponse response) throws IOException {

        // Acquire all of the data needed for a team.
        String cohortID = request.getParameter("cohortID");
        String teamId = request.getParameter("teamId");
        String email = request.getParameter("email");
        
       // You can add a constructor that only requires the email and teamId.
        userDataStore.storeUser(new User(email, teamId);

    }
}
```